huaweicloud/utils/tags.go

```diff
@@ -51,6 +52,38 @@ func UpdateResourceTags(conn *golangsdk.ServiceClient, d *schema.ResourceData, r
 	return nil
 }
 
+// UpdateResourceTagsWithSleep is a helper to update the tags for a resource.
+// It expects the tags field to be named "tags"
+func UpdateResourceTagsWithSleep(conn *golangsdk.ServiceClient, d *schema.ResourceData, resourceType, id string) error {
```

Comment:
Maybe the issue is only happened in MRS, so I think we can just define the function in the package which has the same issue.
If we want to define the function in utils package, I think we can make it more extensional, such as set the sleep times as an input variable.

Reply:
OK，done